zhaquirks/candeo/rotary_dimmer_switches.py

```diff
@@ -0,0 +1,118 @@
+"""Candeo rotary dimmer switches."""
+
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.zcl import ClusterType
+from zigpy.zcl.clusters.general import Identify, Ota
+
+from zhaquirks.candeo import (
+    CANDEO,
+    CandeoLevelControlRemoteCluster,
+    CandeoOnOffRemoteCluster,
+)
+from zhaquirks.const import (
+    CLUSTER_ID,
+    COMMAND,
+    COMMAND_CONTINUED_ROTATING,
+    COMMAND_DOUBLE,
+    COMMAND_HOLD,
+    COMMAND_PRESS,
+    COMMAND_RELEASE,
+    COMMAND_STARTED_ROTATING,
+    COMMAND_STOPPED_ROTATING,
+    CONTINUED_ROTATING,
+    DOUBLE_PRESS,
+    ENDPOINT_ID,
+    LEFT,
+    LONG_PRESS,
+    LONG_RELEASE,
+    PARAMS,
+    RIGHT,
+    ROTARY_KNOB,
+    SHORT_PRESS,
+    STARTED_ROTATING,
+    STOPPED_ROTATING,
+)
+
+remote_quirk = (
+    QuirkBuilder()
+    .replaces(CandeoOnOffRemoteCluster, endpoint_id=2, cluster_type=ClusterType.Client)
+    .replaces(
+        CandeoLevelControlRemoteCluster, endpoint_id=2, cluster_type=ClusterType.Client
+    )
```

Comment:
I'll apply this myself, just want pre-commit to split these into multiple lines.
```suggestion
    .replaces(CandeoOnOffRemoteCluster, endpoint_id=2, cluster_type=ClusterType.Client,)
    .replaces(
        CandeoLevelControlRemoteCluster, endpoint_id=2, cluster_type=ClusterType.Client,
    )
```